keras_nlp/src/utils/transformers/safetensor_utils.py

```diff
@@ -58,7 +65,10 @@ def get_tensor(self, hf_weight_key):
             )
             self.safetensor_files[fname] = file
 
-        return file.get_tensor(hf_weight_key)
+        keys = file.keys()
+        full_key = self.get_prefix(hf_weight_key, keys)
```

Comment:
I think this would not work for the sharded case right? In the sharded case is the prefix also in the  `safetensor_config["weight_map"]`?

Reply:
Yeah. I have updated the code for sharded case.